packages/fluentui/react-northstar/src/components/ItemLayout/ItemLayout.tsx

```diff
@@ -34,9 +34,9 @@ export interface ItemLayoutProps extends UIComponentProps, ContentComponentProps
   endMedia?: any;
   headerMedia?: any;
   media?: any;
-  renderContentArea?: (props: ItemLayoutProps, state: any, classes: ComponentSlotClasses) => React.ReactNode;
```

Comment:
This looks like a breaking change, please mention this.

Reply:
It wasn't used at all anyway before, should I still mention?

Reply:
Mentioned